<launch>
    <node name="load_pcl_list" pkg="pcl_tools" type="load_pcl_list" output="screen" >
        <param name="pcl2_output" type="string" value="/loaded_pcl2" />
        <param name="folderpath" type="string" value="$(find pcl_tools)/doc/examples/load_list/" />
        <param name="filename" type="string" value="test" />
        <param name="tf_extension" type="string" value="pose" />
        <param name="pcl_extension" type="string" value="pcd" />
        <param name="point_cloud_frame_id" type="string" value="/camera" />
        <param name="world_frame_id" type="string" value="/world" />
        <param name="rate" type="double" value="5" />
        <param name="loop" type="bool" value="true" />
    </node>

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find pcl_tools)/rviz/load_point_cloud_list.rviz" />
</launch>
